module/VuFindTheme/src/VuFindTheme/ResourceContainer.php

```diff
@@ -129,12 +129,130 @@ public function addCss($css)
      */
     public function addJs($js)
     {
-        if (!is_array($js) && !is_a($js, 'Traversable')) {
-            $js = [$js];
+        if ((!is_array($js) && !is_a($js, 'Traversable')) || isset($js['file'])) {
+            $this->addJsEntry($js);
+        } elseif (isset($js[0])) {
+            foreach ($js as $current) {
+                $this->addJsEntry($current);
+            }
+        } elseif ($js == []) {
```

Comment:
Is this intentionally not `===`?

Reply:
addJs also accepts strings as parameter; however this should be covered on line 132, so i guess changing it to === shouldn't be a problem to make it clearer.

Reply:
Seems like the current Travis test seems to have a problem with it now. This could be caused by the condition in 132 not matching because strings seem to be Traversable... so either this should be changed back to `==` or the Traversable condition needs to be changed (or we explicitly need to check for strings).

Reply:
No, the problem is that my IDE gave me bad advice, and I should have left that else clause alone! I've just put it back the way it was, and hopefully the tests will pass again. Sorry about that -- dumb mistake on my part. :-)